### ISSUE TITLE
Typed property DOMDocument::$documentElement must not be accessed before initialization

### ISSUE THREAD

**Issue by arifurrahmansw** [tijsverkoyen/CssToInlineStyles#241]
![image](https://user-images.githubusercontent.com/17975092/215408908-c621a026-36d4-4b0d-8d25-91f805881ba8.png)

**Comment by stof** [comment]
Please provide a reproducer (and ideally one that is built with only the library, not a Laravel app where the usage of the library is hidden in tens of abstraction levels).

**Comment by stof** [comment]
And can you tell which exact version of PHP you are using ? This might be a bug in ext-dom.

**Comment by arifurrahmansw** [comment]
Thanks for the quick response. PHP 8.1

**Comment by arifurrahmansw** [comment]
PHP 8.1.0  and Laravel Framework 8.83.27

**Comment by arifurrahmansw** [comment]
when I upgrade laravel 6 to laravel 8 showing this error.

**Comment by stof** [comment]
Please upgrade to the latest 8.1.x release to see whether it fixes the issue. PHP has fixed many issues between 8.1.0 and 8.1.14

**Comment by fakeheal** [comment]
I am having the same problem, my setup includes:
- php: 8.1.17
- tijsverkoyen/css-to-inline-styles: 2.2.6
- laravel/framework: 9.52.7

> PHP has fixed many issues between 8.1.0 and 8.1.14

What changes of php are releated to this package?

**Comment by stof** [comment]
Please try to build a standalone reproducer for this issue. Otherwise, there is nothing I can do to debug the issue.

**Comment by fakeheal** [comment]
> Please try to build a standalone reproducer for this issue. Otherwise, there is nothing I can do to debug the issue.

Yeah, sorry, I thought you might be able to suggest a possible issue off the top of your head, since you mentioned issues in php's versions. I believe it'd be really hard to create a standalone reproducer of the issue, because as you said in Laravel "usage of the library is hidden in tens of abstraction levels".

Thank you for your time!

**Comment by stof** [comment]
Well, some of the PHP 8.1.x releases fixed issues related to typed properties in internal classes. But I'm not sure whether this is before or after 8.1.11.
